docs/api/01-plan/suggestion-buttons/PLAN.md

```
@@ -0,0 +1,426 @@
+1. **新建 `SuggestionButtons` 组件**：
+```tsx
+interface SuggestionButtonsProps {
+  suggestions: string[];
+  onSuggestionClick: (text: string) => void;
+  disabled?: boolean;  // 当正在 streaming 时禁用
+}
+```
+
+2. **渲染位置**：在 `MessageBubble` 中，action buttons 行**上方**、widget deck **下方**（约 MessageList.tsx 第 799 行之后，第 807 行之前）：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)